processing/src/main/java/org/apache/druid/frame/write/FrameWriterUtils.java

```diff
@@ -236,7 +235,7 @@ public static void verifySortColumns(
     for (final KeyColumn keyColumn : keyColumns) {
       final ColumnType columnType = signature.getColumnType(keyColumn.columnName()).orElse(null);
 
-      if (columnType == null || !FieldReaders.create(keyColumn.columnName(), columnType).isComparable()) {
+      if (columnType == null) {
         throw new IAE("Sort column [%s] is not comparable (type = %s)", keyColumn.columnName(), columnType);
```

Comment:
The error message is a little off now. It should say something like: `Sort column[%s] type is unknown`